docs/guides/video-tracks.md

```diff
@@ -0,0 +1,70 @@
+# Video Tracks
+
+Video Tracks are a function of HTML5 video for providing time triggered text to the viewer. Video.js makes video tracks work across all browsers. There are currently six types of tracks:
```

Comment:
good catch on these, copy pasted the format of text-track but never updated this section